### ISSUE TITLE
Error when using with typescript app

### ISSUE THREAD

**Issue by ydeshayes** [feathersjs-ecosystem/feathers-authentication-management#192]
### Steps to reproduce

- [ ] Tell us what broke. The more detailed the better.
- [ ] Include the `feathers-gen-specs.json` file from your app.
- [ ] Include the `src/services/[serviceName]/[serviceName].schema.?s` files if the issue involves the fields in one or more services.

These last 2 items usually allow us to regen enough of your app to recreate the issue.
We may otherwise ask you to provide a minimal GitHub repo or gist isolating the issue.

### Expected behavior
The test should run

### Actual behavior
The test are not running and fail with this error:
```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/myhome/Documents/projects/vtx/server/test/app.test.ts
    at new NodeError (node:internal/errors:371:5)
    at Object.file: (node:internal/modules/esm/get_format:72:15)
    at defaultGetFormat (node:internal/modules/esm/get_format:85:38)
    at defaultLoad (node:internal/modules/esm/load:13:42)
    at ESMLoader.load (node:internal/modules/esm/loader:303:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:230:58)
    at new ModuleJob (node:internal/modules/esm/module_job:63:26)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:244:11)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
    at async formattedImport (/Users/myhome/Documents/projects/vtx/server/node_modules/mocha/lib/nodejs/esm-utils.js:7:14)
    at async Object.exports.requireOrImport (/Users/myhome/Documents/projects/vtx/server/node_modules/mocha/lib/nodejs/esm-utils.js:38:28)
    at async Object.exports.loadFilesAsync (/Users/myhome/Documents/projects/vtx/server/node_modules/mocha/lib/nodejs/esm-utils.js:91:20)
    at async singleRun (/Users/myhome/Documents/projects/vtx/server/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at async Object.exports.handler (/Users/myhome/Documents/projects/vtx/server/node_modules/mocha/lib/cli/run.js:370:5)
```
### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
 "feathers-authentication-management": "^4.0.1",

**NodeJS version**:
v16.13.0

**Operating System**:
Macos

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by fratzinger** [comment]
Sorry, it's hard to tell what the problem is with `Unknown file extension ".ts" for /Users/myhome/Documents/projects/vtx/server/test/app.test.ts`.
Please share more information.

**Comment by ydeshayes** [comment]
I'm still trying to figure it out, but basically, if I have the feathers-authentication-management in my services app.configure(...) I have this error.

**Comment by fratzinger** [comment]
I published a new version `4.0.2`. Please try upgrading and let me know, if your problem still exists.

**Comment by ydeshayes** [comment]
It is working with the new version! thanks a lot!
